src/app/shared/carousel/carousel.html

```diff
@@ -0,0 +1,14 @@
+<div class="docs-carousel-content-wrapper" #contentWrapper (window:resize)="onResize()"
+     [style.minWidth]="shiftWidth + 'px'">
+  <button mat-mini-fab class="docs-carousel-nav docs-carousel-nav-prev" (click)="previous()" *ngIf="this.showPrevArrow">
+    <mat-icon>navigate_before</mat-icon>
+  </button>
+  <div class="docs-carousel-content">
```

Comment:
Maybe add left + right arrow keyboard support as done [here](https://inclusive-components.design/a-content-slider/#keyboardsupport).
```suggestion
  <div class="docs-carousel-content" tabindex="0" (keydown.arrowright)="next()" (keydown.arrowleft)="previous()">
```